app/scripts/modules/kubernetes/container/configurer.directive.html

```diff
@@ -9,53 +9,127 @@
     </div>
   </div>
   <div class="form-group">
-    <div class="col-md-4 sm-label-right">
-      Name
-      <help-field key="kubernetes.containers.name"></help-field>
+    <div class="col-md-3 sm-label-right">
+      Registry 
+      <help-field key="kubernetes.containers.registry"></help-field>
     </div>
     <div class="col-md-6">
-      <input type="text" class="form-control input-sm" name="details" ng-model="container.name"/>
+      <p>{{container.registry}}</p>
```

Comment:
You may (or may not) get slightly better alignment with the label if you change this to:
```html
<p class="form-control-static">{{container.registry}}</p>
```